middlewares/validators/user.js

```diff
@@ -192,10 +193,31 @@ async function validateUserQueryParams(req, res, next) {
   }
 }
 
+/**
+ * Validator function for query params for the filter route
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ * @param next {Object} - Express middleware function
+ */
+const validateImageVerificationQuery = async (req, res, next) => {
+  const { type: imageType } = req.query;
+  try {
+    if (!IMAGE_VERIFICATION_TYPES.includes(imageType)) {
+      throw new Error("Invalid verification type was provided!");
+    }
+    next();
```

Comment:
Joi has a xor function that lets you check if anyone one of the two things is provided, using that might save us the trouble of writing a throwing error. Please take an opinion from @heyrandhir he has worked with it.